markdown/extensions/md_in_html.py

```diff
@@ -42,7 +42,7 @@ class HTMLExtractorExtra(HTMLExtractor):
 
     def __init__(self, md: Markdown, *args, **kwargs):
         # All block-level tags.
-        self.block_level_tags = set(md.block_level_elements.copy())
+        self.block_level_tags = md.block_level_elements.copy()
```

Comment:
I just noticed this, but this was already being converted to a set in this extension. Maybe we should keep that behavior rather than leave it as our custom type.

Reply:
Right, I removed `set()` around the copy because it was redundant when our block level elements were a plain set, but now it's a hybrid container with a custom `copy` method, we should wrap again :+1:

Reply:
Done. I removed the call to `.copy()` since calling `set` on the object creates a new object anyway, and the container is "single-level" (only strings, no references to other mutable structs).